README.md

```diff
@@ -157,6 +157,64 @@ datastore.update({
 });
 ```
 
+###  scan
+
+Scan a record in the datastore. Returns `[]` if the record is empty. Returns error if the `tableName` does not exist.
```

Comment:
`Scan a table in the datastore. Return [] if the table is empty. Returns error if the table does not exist`